Classes/BITHockeyManager.h

```diff
@@ -375,6 +375,15 @@
 /// @name Environment
 ///-----------------------------------------------------------------------------
 
+
+/**
+ Enum that determines what kind of environment the application is installed and running in.
+ 
+ See `BITEnvironment`
+ */
+@property (nonatomic, readonly) BITEnvironment appEnvironment;
```

Comment:
@anlinde @chrwend Suggestions to improve the documentation of this new property?